### ISSUE TITLE
Use warnings.warn in tiff_file.py: get_values instead of stderr

### ISSUE THREAD

**Issue by moi90** [pearu/pylibtiff#42]
I may have non-standard tiff files. Loading them `tiff.TIFFfile` writes an error to stderr, but the result is usable nevertheless.

```
get_values: incomplete info for type=7 ['7']: dtype=None, bytes=1
```

I expect pylibtiff to use warnings.warn instead, so that the user may handle such cases.

**Comment by moi90** [comment]
I created a pull request: #43

**Comment by moi90** [comment]
Are contributions like this unwanted?

**Comment by pearu** [comment]
No. Such contributions, especially pull requests, are highly appreciated.
There was some time period where my github messaging was disabled (also in other projects) and so I missed many messages.
Thanks again for your contribution!

It seems that pull request #43 got a conflict with previous Python 3 support merges..
